includes/class-amp-theme-support.php

```diff
@@ -2107,6 +2096,15 @@ private static function get_optimizer_configuration( $args ) {
 		 */
 		$enable_ssr = apply_filters( 'amp_enable_ssr', $enable_ssr );
 
+		/**
+		 * Filter whether the AMP Optimizer should use ES modules for the runtime and extensions.
+		 *
+		 * @since 2.1.0
+		 *
+		 * @param bool $enable_esm Whether the AMP Optimizer should use ES modules for the runtime and extensions.
+		 */
+		$enable_esm = apply_filters( 'amp_enable_esm', $enable_esm );
+
```

Comment:
```suggestion
```

Reply:
Can you edit this one to include one of the blank lines? Otherwise we'll end up with 2 of them...

Reply:
Done in 3d1175a0a